arkade/src/commonMain/kotlin/com/arkade/core/services/BatchManagementService.kt

```
@@ -0,0 +1,94 @@
+            if (intent == null || activeBatchSessions.containsKey(intentId)) {
+                return@forEach
+            }
+
+            try {
+            } catch (e: Exception) {
+                Log.warning(LOG_TAG, "Failed to handle batch started event for intent $intentId: $e")
+            }
+        }
+    }
+
+    private suspend fun setupBatchSession(event: BatchEvent.BatchStartedEvent) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)